server/app.ts

```diff
@@ -28,18 +29,26 @@ let plugins: {
 	[name: string]: Notifier<any>;
 } = {};
 
-const resolvers = {
+type Ctx = express.Request;
+
+interface IResolver {
+	Query: types.Query<Ctx>;
+}
+
+const resolvers: IResolver = {
 	Query: {
-		send_message: async (prev: any, args: any): Promise<IPluginReturn[]> => {
+		send_message: async (prev, args): Promise<IPluginReturn[]> => {
 			const message = args.message;
-
-			const checkQueue = Object.keys(args.plugins).map( rawName => {
+			const graphQLPlugins = args.plugins as {[name: string]: any}; // Help! typewriter doesn't know these are strings!
```

Comment:
?

Reply:
See comment, I get an error along the lines of `args.plugins` has no index type because corresponding
```
 export interface PluginMaster {
        live_site?: LiveSiteConfig
        slack?: SlackConfig
        twitter?: TwitterConfig
    }
```
has no index type, and verifiedConfig would therefore be `any` (which ts isn't happy about)